server/executor/poller_executor_test.go

```diff
@@ -0,0 +1,66 @@
+package executor_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/kubeshop/tracetest/server/config"
+	"github.com/kubeshop/tracetest/server/executor"
+	"github.com/kubeshop/tracetest/server/model"
+	"github.com/kubeshop/tracetest/server/testdb"
+	"github.com/kubeshop/tracetest/server/tracedb"
+	"github.com/kubeshop/tracetest/server/tracing"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_PollerExecutor_ExecuteRequest(t *testing.T) {
+	t.Parallel()
+
+	// Cases: tests with 0 traces
+	// Cases: tests with only 1 trace
+	// Cases: tests with more than 1 trace
```

Comment:
with 0 `spans`? right?

Reply:
Yes! I'm still finishing this test, hopefully, I'll enable it for review at the end of the day